plugins/clipboard-manager/guest-js/index.ts

```diff
@@ -53,4 +54,60 @@ async function readText(): Promise<string> {
   return kind.plainText.text;
 }
 
-export { writeText, readText };
+/**
+ * Writes HTML or fallbacks to write provided plain text to the clipboard.
+ * @example
+ * ```typescript
+ * import { writeHtml, readHtml } from '@tauri-apps/plugin-clipboard-manager';
+ * await writeHtml('<h1>Tauri is awesome!</h1>', 'plaintext');
+ * await writeHtml('<h1>Tauri is awesome!</h1>', '<h1>Tauri is awesome</h1>'); // Will write "<h1>Tauri is awesome</h1>" as plain text
+ * assert(await readHtml(), '<h1>Tauri is awesome!</h1>');
+ * ```
+ *
+ * @returns A promise indicating the success or failure of the operation.
+ *
+ * @since 2.0.0
+ */
+async function writeHtml(
+    html: string,
+    alt_html: string
+): Promise<void> {
+  return invoke("plugin:clipboard-manager|write_html", {
+    data: {
+      html: {
+        html,
+        alt_html
+      },
+    },
+  });
+}
```

Comment:
we should make this optional, also use camelCase for JS APIs
```suggestion
async function writeHtml(
    html: string,
    altHtml?: string
): Promise<void> {
  return invoke("plugin:clipboard-manager|write_html", {
    data: {
      html: {
        html,
        altHtml
      },
    },
  });
}
```